docs/fsharp/tour.md

```diff
@@ -1,4 +1,158 @@
+
+[!code-fsharp[BasicFunctions](../../samples/snippets/fsharp/tour.fs#L25-L51)]
+
+`let` bindings are also how you bind a value to a name, similar to a variable in other languages.  However, unlike language such as C#, these values are immutable by default.  The lines between values and functions are a bit blurred, and that's intentional.  More on that later.
+
+## Integers, Booleans, and Strings
+
+As a .NET language, F# supports the same underlying [primitive types](language-reference/primitive-types.md) that exist in .NET.
+
+Here's some Integer values, generated in different ways:
+
+[!code-fsharp[Integers](../../samples/snippets/fsharp/tour.fs#L8-L22)]
+
+Here's what Boolean values and performing basic conditional logic looks like:
+
+[!code-fsharp[Bools](../../samples/snippets/fsharp/tour.fs#L59-L66)]
+
+And here's what basic [string](language-reference/strings.md) manipulation looks like:
+
+[!code-fsharp[Strings](../../samples/snippets/fsharp/tour.fs#L74-L90)]
+
+## Tuples
+
+[Tuples](language-reference/tuples.md) are a big deal in F#.  They are a grouping of unnamed, but ordered values, that can be treated as values themselves.  Think of them as values which are aggregated from other values.  They have many uses, such as conveniently returning multiple values from a function, or grouping values for some ad-hoc convenience.
+
+[!code-fsharp[Tuples](../../samples/snippets/fsharp/tour.fs#L98-L112)]
+
+You can also use *struct tuples* with the `struct` keyword.  This tells the compiler that the tuple is allocated on the program Stack, rather than the program Heap.  This is helpful for some high-performance scenarios.
+
+[!code-fsharp[Tuples](../../samples/snippets/fsharp/tour.fs#L114-L121)]
+
+Note that struct tuples and "regular" tuples are distinct types.  You cannot "fool" the compiler into thinking one is the other.  Instead, you must explicitly transfer the values from one tuple type to the other if you wish to work with both types of tuples.
+
+## Arrays, Lists, and Sequences
+
+Arrays, Lists, and Sequences are three primary collection types in the F# core library.
+
+[Arrays](language-reference/arrays.md) are fixed-size, *mutable* collections of elements of the same type.  They are also all loaded into memory, thus they support fast random access of elements.
+
+[!code-fsharp[Arrays](../../samples/snippets/fsharp/tour.fs#L246-L276)]
+
+[Lists](language-reference/lists.md) are ordered, immutable collections of elements of the same type.  Under the covers, they are singly-linked lists, which makes them lightweight and good for enumeration, but a poor choice for random access and concatenation if they're large.
```

Comment:
Not sure "under the covers" is appropriate here, it's not an implementation detail, that's literally the public definition of `List<'T>`.

Also, lists are lightweight and good for enumeration when compared with what? Certainly not arrays, those are more lightweight and enumerating them is also more efficient.